<?xml version="1.0"?>

<launch>

    <arg name="trigger_topic_name" default="/scout/mavros/state" />
    <arg name="record_script" default="$(find kaist_rosbag)/config/scout_rosbag.sh" />
    <arg name="record_folder" default="" />
    <arg name="verbose" default="false" />

    <node name="kaist_rosbag" pkg="kaist_rosbag" type="kaist_rosbag.py" output="screen">
        <param name="record_script" value="$(arg record_script)" />
        <param name="record_folder" value="$(find kaist_rosbag)/bags/$(arg record_folder)" />
        <param name="trigger_topic_name" value="$(arg trigger_topic_name)" />
        <param name="verbose" value="$(arg verbose)" />
    </node>

</launch>
